templates/osync.conf

```diff
@@ -0,0 +1,68 @@
+INSTANCE_ID="sysvol_sync"
+INITIATOR_SYNC_DIR="/var/lib/samba/sysvol"
+TARGET_SYNC_DIR="ssh://{{samba_sysvol_sync_target_user}}@{{item}}//var/lib/samba/sysvol"
+SSH_RSA_PRIVATE_KEY="{{samba_sysvol_sync_ssh_key}}"
+SSH_PASSWORD_FILE=""
+#_REMOTE_TOKEN=SomeAlphaNumericToken9
+CREATE_DIRS=yes
+LOGFILE=""
+MINIMUM_SPACE=10240
+BANDWIDTH=0
+SUDO_EXEC=no
+RSYNC_EXECUTABLE=rsync
+RSYNC_REMOTE_PATH=""
+RSYNC_PATTERN_FIRST=include
+RSYNC_INCLUDE_PATTERN=""
+RSYNC_EXCLUDE_PATTERN=""
+RSYNC_INCLUDE_FROM=""
+RSYNC_EXCLUDE_FROM=""
+PATH_SEPARATOR_CHAR=";"
+SSH_COMPRESSION=yes
+SSH_IGNORE_KNOWN_HOSTS=no
+REMOTE_HOST_PING=no
+REMOTE_3RD_PARTY_HOSTS=""
+RSYNC_OPTIONAL_ARGS=""
+PRESERVE_PERMISSIONS=yes
+PRESERVE_OWNER=yes
+PRESERVE_GROUP=yes
+PRESERVE_EXECUTABILITY=yes
+PRESERVE_ACL=yes
+PRESERVE_XATTR=yes
+COPY_SYMLINKS=no
+KEEP_DIRLINKS=no
+PRESERVE_HARDLINKS=no
+CHECKSUM=no
+RSYNC_COMPRESS=yes
+SOFT_MAX_EXEC_TIME=7200
+HARD_MAX_EXEC_TIME=10600
+KEEP_LOGGING=1801
+MIN_WAIT=60
+MAX_WAIT=7200
+CONFLICT_BACKUP=yes
+CONFLICT_BACKUP_MULTIPLE=no
+CONFLICT_BACKUP_DAYS=30
+CONFLICT_PREVALANCE=initiator
+SOFT_DELETE=yes
+SOFT_DELETE_DAYS=30
+SKIP_DELETION=initiator
+RESUME_SYNC=yes
+RESUME_TRY=2
+FORCE_STRANGER_LOCK_RESUME=no
+PARTIAL=no
+DELTA_COPIES=yes
+DESTINATION_MAILS="your@alert.tld"
+MAIL_BODY_CHARSET=""
+SENDER_MAIL="alert@your.system.tld"
+SMTP_SERVER=smtp.your.isp.tld
+SMTP_PORT=25
+SMTP_ENCRYPTION=none
+SMTP_USER=
+SMTP_PASSWORD=
```

Comment:
Are the 8 lines above (which apply to sending something by post) mandatory? Without them, osync does not work?
It’s just that even the .tld domains specified in any parameters are a bad idea if this could be avoided.

Reply:
deleted